modules/_labs/header/react/lib/parts/SearchBar.tsx

```diff
@@ -145,35 +149,49 @@ const SearchForm = styled('form')<
   }
 );
 
-const SearchContainer = styled('div')({
-  position: `relative`,
-  width: `100%`,
-  textAlign: 'left',
-  minHeight: height,
-  height: height, // Needed to keep IE11 vertically centered
-});
+const SearchContainer = styled('div')<Pick<SearchBarProps, 'height'>>(
+  {
+    position: `relative`,
+    display: 'flex',
+    alignItems: 'center',
+    width: `100%`,
+    textAlign: 'left',
+  },
+  ({height}) => ({
+    minHeight: height,
+    height: height, // Needed to keep IE11 vertically centered
+  })
+);
 
 const SearchCombobox = styled(Combobox)({
   width: `100%`,
 });
 
 const SearchIcon = styled(IconButton)<Pick<SearchBarProps, 'isCollapsed'> & {isHidden: boolean}>(
   ({isCollapsed, isHidden}) => {
+    const collapsedSize = 40;
+    const size = 32;
     const collapseStyles: CSSObject = isCollapsed
       ? {
-          width: spacing.xl,
-          height: spacing.xl,
+          minWidth: collapsedSize,
+          width: collapsedSize,
+          minHeight: collapsedSize,
+          height: collapsedSize,
```

Comment:
I'm avoiding using `spacing` values for height and width measurements. Just because `spacing.xl` = 40 pixels doesn't mean that for every CSS value that uses 40px we need to use `spacing.xl`

Reply:
We will eventually want to migrate _everything_ (including widths and heights) to use `spacing`. We will ultimately move towards `rems` (like we are with typography), and it would be weird if the whole page scaled (typography, spacing, sizes, etc.) except for a few elements.

Not a big deal for this to go in as is though, because we'll have to do a serious audit for this in the future anyway.